web/client/components/style/ThemaClassesEditor.jsx

```diff
@@ -40,16 +39,13 @@ class ThemaClassesEditor extends React.Component {
                 </Col>
                 <Col xs="4">
                     <NumberPicker
-                        format="- ###.###"
```

Comment:
So we lost the ability to specify a format... we can live without it, I think

Reply:
Yes, for the moment I couldn't implement all the functionalities of the original one.